### PR TITLE
Fix nox testing so that MSDplus is installed and tests are run.

### PR DESCRIPTION
**Pull request by kuchtact**

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 def tests(session):
     """Run tests with pytest."""
     session.install(".")
-    session.run(pytest)
+    session.run("pytest")
```
Comment (namurphy) [inline noxfile.py]: Good catch!

**Comment by kuchtact** [comment]
@broantognetti When you installed MDSplus on your windows computer did you do it through the WSL terminal or did you run it in command prompt? It looks like MDSplus doesn't have a windows conda package so that's why I'm wondering.

**Comment by kuchtact** [comment]
For the sake of getting this pull request done, I think it's ready for review. Eventually setting up WSL in the GitHub Action would be nice so that we can test that everything works on that side of the world but if we are just using mamba anyhow through Linux (but on Windows) I would hope that it would be exactly the same.

**Comment by broantognetti** [comment]
> @broantognetti When you installed MDSplus on your windows computer did you do it through the WSL terminal or did you run it in command prompt? It looks like MDSplus doesn't have a windows conda package so that's why I'm wondering.

I use WSL2 on my windows PC which means I essentially have a separate Linux kernel running within a virtual machine. So for that I installed MDSPlus in the command prompt of my Ubuntu OS within the Windows VM, WSL may be slightly different in how it operates though.

**Comment by kuchtact** [comment]
Ok. Looking online it seems like running stuff on WSL2 isn't all that feasible with github actions so maybe for now we can safely ignore Windows testing unless we want to test on WSL1. I'm guessing most people nowadays use WSL2 and if the tests run on Linux then maybe we don't need to test it. I think I need to fix the pre-commit stuff before merging so let me do that first before reviewing.

**Comment by kuchtact** [comment]
This pull request closes #13.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -33,11 +33,6 @@ jobs:
           python: '3.11'
           nox_session: tests-3.11
 
-        - name: Tests, Python 3.10, Windows
```
Comment (broantognetti) [inline .github/workflows/ci.yml:36]: bleh, windows

**Review by broantognetti — APPROVED** [review]

**Review by ciortiz — APPROVED** [review]
Looks like installation is set up for MacOS, Linux and WSL, and to be done later for Windows (which is presumably not used without WSL, anyway). Looks good!

**Pull request merged**